rpcd/playbooks/roles/beaver/templates/beaver.j2

```diff
@@ -37,6 +37,7 @@ do_start() {
 }
 
 do_stop() {
+    test -d "$( dirname $BEAVER_PID )" || mkdir "$( dirname $BEAVER_PID )" && chown $BEAVER_NAME:$BEAVER_NAME "$( dirname $BEAVER_PID )"
```

Comment:
shouldn't this be in start, not stop?

also, doing it all on one line is generally bad practice, can you fix that?

    local BEAVER_PID_DIRNAME="$(dirname $BEAVER_PID)"
    if [[ ! -d $BEAVER_PID_DIRNAME ]]; then
        mkdir -p "${BEAVER_PID_DIRNAME}"
        chown "${BEAVER_NAME}":"${BEAVER_NAME}"
    fi